packages/web3-shared/base/src/utils/resolver.ts

```diff
@@ -125,11 +125,13 @@ export function createExplorerResolver<ChainId, SchemaType, NetworkType>(
                 domain,
                 ...getExplorerURL(chainId)?.parameters,
             }),
-        fungibleTokenLink: (chainId: ChainId, address: string) =>
-            urlcat(getExplorerURL(chainId).url, fungibleTokenPathname, {
+        fungibleTokenLink: (chainId: ChainId, address: string) => {
+            if (!chainId || !address) return ''
```

Comment:
It doesn't make sense. Why even `chainId` could be not given?

Reply:
In https://github.com/DimensionDev/Maskbook/issues/7516 , the address is empty string. But failed to reproduce it.

Reply:
I see it claimed that: `Error: Path parameter address cannot be an empty string.` Not caused by `chainId`, right?